docs/faq.md

```diff
@@ -188,7 +188,7 @@ will return the wire error.
 Alternatively, if you wish to cancel the operation and immediately stop
 the client stream, see [below](#cancel-stream) to cancel the operation.
 
-### How do I cancel a client response stream in connect-go?
+### How do I cancel a client response stream in Connect-go?
```

Comment:
If we are bothering to update this, I thought we decided on `Connect-Go`

https://bufprivate.slack.com/archives/C04QDA286JJ/p1706661038722099?thread_ts=1706635492.240409&cid=C04QDA286JJ